app/models/User.py

```diff
@@ -0,0 +1,12 @@
+
+from app import db
+
+class User(db.Model):
```

Comment:
can we add more common fields o user schema?

Reply:
Suggestions?

Reply:
Like password(encrpted), first name, last name, phone etc